protocol-designer/src/organisms/Alerts/FormAlerts.tsx

```diff
@@ -70,6 +77,8 @@ function FormAlertsComponent(props: FormAlertsProps): JSX.Element | null {
     focusedField,
     dirtyFields: dirtyFields ?? [],
     errors: formLevelErrorsForUnsavedForm,
+    page: page,
```

Comment:
nit
same name so can remove `: page`
```suggestion
    page
```

Reply:
question about page type.
visibleFormErrors: StepFormErrors and StepFormErrors = FormError[]
page is optional in FormError.
Does page need to be non-optional?